### ISSUE TITLE
Definition name cannot be null or empty  

### ISSUE THREAD

**Issue by theseer** [MontealegreLuis/phuml#13]
When giving phuml a run against Templado (https://templado.io), phuml crashes with the following error:

```
theseer@nyda /tmp/x5 $ php ./tools/phuml phuml:diagram -r -p neato  ~/storage/php/templado/engine/src test.png
[|] Running... (This may take some time)
[|] Parsing codebase structure

In Assert.php line 2060:
                                           
  Definition name cannot be null or empty  

```                                           

Not sure what to do here next :)

**Comment by MontealegreLuis** [comment]
Hi @theseer the problem is that using `object` in a DockBlock will return an `Object_` but with `null` value.

I have fixed it in this MR https://github.com/MontealegreLuis/phuml/pull/15 if you want to give it a try.

I checked locally and I can generated diagrams for both Templado and Phive.

I found another issue after generating those diagrams with definitions like this one:

```php
namespace PharIo\Phive;

class EnvironmentException extends \Exception implements Exception {
}
```

I usually alias the class instead of using the fully qualified name.

I'll open another MR with a fix for that
